packages/lending/src/protocols/radiant-v2/lending-protocol.test.ts

```diff
@@ -198,149 +197,252 @@ describe('Test Radiant V2 LendingProtocol', function () {
       {
         chainId: common.ChainId.bnb,
         account: '0xA15aEF397F9Dc9dE23C8e3Df8A289B21D4cA8fCD',
-        blockTag: 38738990,
+        blockTag: 40750000,
         expected: {
           chainId: 56,
           protocolId: 'radiant-v2',
           marketId: 'bnb',
           utilization: '1',
-          healthRate: '1.03627738230590567017',
-          totalSupplyUSD: '7113376.792946228696130057217',
-          totalBorrowUSD: '5148266.94649033938170577969',
+          healthRate: '1.04397149107568892179',
+          netAPY: '-0.14511936117721748453',
+          totalSupplyUSD: '5420653.9101024398073132710459',
+          totalBorrowUSD: '3894254.2658896485547761754501',
           supplies: [
             {
-              token: bnbTokens.BTCB,
-              price: '62129.33',
-              balance: '114.493054935484124489',
-              apy: '0.0018842556485471865',
+              token: {
+                chainId: 56,
+                address: '0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c',
+                decimals: 18,
+                symbol: 'BTCB',
+                name: 'BTCB Token',
+                logoUri: 'https://cdn.furucombo.app/assets/img/token/WBTC.svg',
```

Comment:
Do you think it's better to get the token information from other file/package like the old one OR it's better to have a fixed value here?

Reply:
Directly using fixed values might be more efficient, tests might still need to be updated manually whenever new tokens are added